syft/distro/distro.go

```diff
@@ -8,23 +8,28 @@ import (
 
 // Distro represents a Linux Distribution.
 type Distro struct {
-	Type       Type
+	Type       string
 	Version    *hashiVer.Version
 	RawVersion string
 	IDLike     string
 }
 
 // NewDistro creates a new Distro object populated with the given values.
-func NewDistro(t Type, ver, like string) (Distro, error) {
+func NewDistro(name, ver, like string) (Distro, error) {
```

Comment:
Same issue here - We pass name, but it's actually NOT the name field from the file we parse. I can rename this to id and then it will match in the function and be less confusing to readers who come back here.

```
ID: id,
```

Reply:
Agreed — we should be consistent here, _especially_ since `name` is a real, separate thing.